tests/modules/programs/nnn/nnn.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+
+{
+  config = {
+    programs.nnn = {
+      enable = true;
+      bookmarks = {
+        d = "~/Documents";
+        D = "~/Downloads";
+        p = "~/Pictures";
+        v = "~/Videos";
+      };
+      extraPackages = with pkgs; [ foo bar ];
+      plugins = {
+        src = ./plugins;
+        mappings = {
+          c = "fzcd";
+          f = "finder";
+          v = "imgview";
+        };
+      };
+    };
+
+    test.stubs = {
+      # TODO: dummy nnn doesn't work here since there is no $out
```

Comment:
I tried either creating `nnnDummy = { outPath = "nnn"; }` or a `pkgs.writeShellScriptBin "nnn" "";` and using it as `config.programs.nnn.package`.

On the first case it didn't work because there where no `$out/bin/nnn`. In the second case `wrapProgram` simply didn't do anything (and the `$out/bin/nnn` stayed with their original contents).

Reply:
But maybe this isn't too much an issue. `nnn` is quite fast to build.

Reply:
The `buildScript` can be changed to create `$out`: `test.stubs.something.buildScript = ''mkdir -p "$out"/bin; touch "$out"/bin/nnn'';`

Reply:
I tried this but for some reason `home-path/bin/nnn` is returning empty.

Reply:
I think the `postInstall` hook is not triggering.

Reply:
Got it to work:

```nix
    test.stubs = {
      nnnDummy.buildScript = ''
        mkdir -p "$out/bin"
        touch "$out/bin/nnn"
        chmod +x "$out/bin/nnn"

        runHook postInstall
      '';
    }
```